### PR TITLE
Add Git books

### PR DESCRIPTION
**Pull request by Mannuel25**
## What does this PR do?
Add resource(s) 

### Why is this valuable (or not)?
It's valuable because, it'll help people know and learn more about how to use git
### How do we know it's really free?
I checked the website that the resources are to confirm it's free.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -790,27 +790,32 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [A git Primer](https://danielmiessler.com/study/git/) - Daniel Miessler
 * [A Visual Git Reference](http://marklodato.github.io/visual-git-guide/index-en.html) - Mark Lodato
 * [Conversational Git](http://blog.anvard.org/conversational-git/)
+* [Distributed Version Control with Git: Mastering the Git command line - Third Edition](https://www.pdfdrive.com/distributed-version-control-with-git-mastering-the-git-command-line-third-edition-e157843022.html) - Lars Vigel & Alex Blewitt
```
Comment (eshellman) [inline books/free-programming-books-langs.md:793]: In general, we do not link to file sharing sites such as pdfdrive, or file hosting sites such as google docs. or dropbox.

File: books/free-programming-books-langs.md [hunk 2]
```diff
@@ -790,27 +790,32 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
...
 * [git - the simple guide](http://rogerdudler.github.io/git-guide/)
 * [Git for Computer Scientists](http://eagain.net/articles/git-for-computer-scientists/)
 * [Git From The Bottom Up](https://jwiegley.github.io/git-from-the-bottom-up/) - J. Wiegley
 * [Git Immersion](http://gitimmersion.com)
 * [Git In The Trenches](http://cbx33.github.io/gitt/index.html)
 * [Git internals](https://github.com/pluralsight/git-internals-pdf/raw/master/drafts/peepcode-git.pdf) - Scott Chacon (PDF)
+* [Git: Learn Version Control with Git: A step-by-step Ultimate beginners Guide](https://www.pdfdrive.com/git-learn-version-control-with-git-a-step-by-step-ultimate-beginners-guide-e158237381.html) - Dennis Hutten (PDF)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:800]: In general, we do not link to file sharing sites such as pdfdrive, or file hosting sites such as google docs. or dropbox.

File: books/free-programming-books-langs.md [hunk 3]
```diff
@@ -790,27 +790,32 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
...
 * [Git Magic](http://www-cs-students.stanford.edu/~blynn/gitmagic/)
-* [Git Notes for Professionals](http://goalkicker.com/GitBook) - Compiled from StackOverflow Documentation (PDF)
 * [Git Pocket Guide](https://www.oreilly.com/library/view/git-pocket-guide/9781449327507) - Richard E. Silverman
 * [Git Reference](https://web.archive.org/web/20170602211147/http://gitref.org/)
 * [Git Succinctly, Syncfusion](https://www.syncfusion.com/resources/techportal/ebooks/git) (PDF, Kindle) (email address *requested*, not required)
+* [Git for Teams: A User-Centered Approach to Creating Efficient Workflows in Git](https://www.pdfdrive.com/git-for-teams-a-user-centered-approach-to-creating-efficient-workflows-in-git-e158737881.html) - Emma Jane &Hogbin Westby (PDF)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:805]: In general, we do not link to file sharing sites such as pdfdrive, or file hosting sites such as google docs. or dropbox.

File: books/free-programming-books-langs.md [hunk 4]
```diff
@@ -790,27 +790,32 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
...
 * [Git-Tutorial For-Beginners](https://product.hubspot.com/blog/git-and-github-tutorial-for-beginners) - HubSpot Product Team
 * [Git Workflows](https://web.archive.org/web/20210910133251/http://documentup.com/skwp/git-workflows-book) - Yan Pritzker *(:card_file_box: archived)*
+* [Git Notes for Professionals](https://goalkicker.com/GitBook) - 
 * [Happy Git and GitHub for the useR](https://happygitwithr.com) - Jenny Bryan
 * [How to Collaborate on Github](https://github.com/eonist/How-to-collaborate-on-github) - André J
 * [Introduction to Git and Github](https://launchschool.com/books/git) - Launch School
 * [Introduction to Git and Github - Tutorial](http://cse.unl.edu/~cbourke/gitTutorial.pdf) - Dr. Chris Bourke (PDF)
 * [Introduction to Git and GitHub eBook](https://github.com/bobbyiliev/introduction-to-git-and-github-ebook) - Bobby Iliev (Markdown, PDF)
 * [Learn Git - Learn Version Control with Git](http://www.git-tower.com/learn/git/ebook/command-line/introduction) - Tobias Günther
+* [Learn Git in a Month of Lunches](https://www.pdfdrive.com/learn-git-in-a-month-of-lunches-e34369237.html) - Rick Umali (PDF)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:816]: In general, we do not link to file sharing sites such as pdfdrive, or file hosting sites such as google docs. or dropbox.

File: books/free-programming-books-langs.md [hunk 5]
```diff
@@ -790,27 +790,32 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
...
 * [Pro Git](http://git-scm.com/book/en/v2) - Scott Chacon
+* [Professional Git](https://www.pdfdrive.com/professional-git-e176044429.html) - Brent Laster
```
Comment (eshellman) [inline books/free-programming-books-langs.md:818]: In general, we do not link to file sharing sites such as pdfdrive, or file hosting sites such as google docs. or dropbox.

**Review by eshellman — COMMENTED** [review]
most files on pdfdrive are not authorized distributions. Please see CONTRIBUTING.md guidelines.

**Comment by Mannuel25** [comment]
Okay, thanks for the feedback 🙏

**Pull request closed without merging**